### DISCUSSION TITLE
hey some one have use aws athena

### DISCUSSION THREAD

**Discussion by michaelr231** [osm-search/Nominatim#2586] [Q&A]
hey someone have any experience using aws athena to query cities polygon on geojson fomrat? i use planet from s3, and i think i need to create another table for polygons city boundary? am i right? with .poly info? or there is some query that i missing?
thanks!

**Comment by mtmail** [comment]
Is the question related to Nominatim? OpenStreetMap has a separate Q&A forum at https://help.openstreetmap.org/ (also https://gis.stackexchange.com/)

**Comment by michaelr231** [comment]
actually yes, because i use Nominatim reverse function and try to fetch the same data from aws athena querys
